specs/xaml-backdrop-api.md

```diff
@@ -0,0 +1,316 @@
+[DispatcherQueue](https://docs.microsoft.com/uwp/api/Windows.System.DispatcherQueue),
+coordinate with the
+[Loaded](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Xaml.FrameworkElement.Loaded)
+event, and check
+[IsSupported](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Composition.SystemBackdrops.MicaController.IsSupported)
+to see if you need to provide a fallback (Mica isn't supported on Windows 10).
+
+So it's a great feature and works, but it's difficult to use.
+[Here's a full code example of the current solution](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/system-backdrop-controller#example-use-mica-in-a-windows-appsdkwinui-3-app).
+
+The purpose of the new APIs here are to make backdrops much easier to use in a Xaml app,
+allowing you to simply set a backdrop as a property on a `Window`.
+
+
+# API Pages
+
+## Window.SystemBackdrop property
+
+_This description also applies to `Popup.SystemBackdrop` and `FlyoutBase.SystemBackdrop` properties_
+
+Sets a `SystemBackdrop` to apply that backdrop to this `Window`.
+
+This backdrop is what will render behind the content specified in `Window.Content`.
+If all of the content is fully opaque, this backdrop will have no visible effect.
+
+The following example creates a Window with a Mica backdrop.
+
+```xml
+<Window
+    x:Class="App3.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" >
+
+    <Window.SystemBackdrop>
+        <MicaBackdrop/>
+    </Window.SystemBackdrop>
+
+    <Grid ColumnDefinitions="Auto,*">
+
+        // This area has a transparent background, so the Mica backdrop will be mostly visible
+        // For example if there are buttons here, the backdrop will show up in
+        // the margins and gaps between the button text
+        <local:NavigationControls/>
+
+        // This area has an opaque background, so the Mica backdrop won't be visible
+        <local:ContentArea Background="White"/>
+
+    </Grid>
+</Window>
+```
+
+
+## MicaBackdrop class
+
+_Same description applies to `DesktopAcrylicBackdrop` class_
+
+_See also the base class: `SystemBackdrop`_
+
+Use this class to set a backdrop on a
+[Window](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Xaml.Window),
+[Popup](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Xaml.Controls.Primitives.Popup),
+or [FlyoutBase](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Xaml.Controls.Primitives.FlyoutBase)
+(such as a `Flyout` or `MenuFlyout`).
+
+For example:
+
+```xml
+<Window x:Class="MyApp.MainWindow">
+
+    <Window.SystemBackdrop>
+        <MicaBackdrop MicaKind="BaseAlt"/>
+    </Window.SystemBackdrop>
+
+</Window>
+```
+
+Note that `MicaBackdrop` isn't supported on all systems.
+In such cases a solid color will be used instead of the Mica effect.
```

Comment:
How is the fallback color picked? Does the end developer specify it somehow?

Reply:
Discussion: how the value is picked currently is unspecified, implementation detail. Plan though is to add a Fallback color property to `MicaBackdrop`.

Reply:
Addressed with the following added info:

"The provided Xaml backdrop materials MicaBackdrop and DesktopAcrylicBackdrop currently do not expose FallbackColor or the material customization properties (TintOpacity, etc), instead relying on the default Light/Dark configurations of the underlying MicaController and DesktopAcrylicController. To customize these properties, createa a custom material class deriving from SystemBackdrop that exposes the desired properties."